packages/babel-plugin/src/utils/traverse-expression/traverse-call-expression.tsx

```diff
@@ -0,0 +1,116 @@
+import * as t from '@babel/types';
+
+import type { Metadata } from '../../types';
+import { getPathOfNode, wrapNodeInIIFE } from '../ast';
+import { createResultPair } from '../create-result-pair';
+import { evaluateExpression } from '../evaluate-expression';
+import { resolveBinding } from '../resolve-binding';
+
+import { traverseMemberExpression } from './traverse-member-expression';
```

Comment:
This is a cyclic import :( `traverseMemberExpression` imports `traverseCallExpression` too.

Reply:
- [ ] resolve cyclic import